README.md

```diff
@@ -7,64 +7,65 @@
 
-* What are the levels(jr, pl, sr) in my context? (for your manager, tribe and company)
-> You can read this [awesome article](https://www.swyx.io/writing/career-ladders/) about engineering career ladders on [swyx.io](https://www.swyx.io/)
+- What are the levels(jr, pl, sr) in my context? (for your manager, tribe and company)
+
+  > You can read this [awesome article](https://www.swyx.io/writing/career-ladders/) about engineering career ladders on [swyx.io](https://www.swyx.io/)
+
+- Where am i?
 
-* Where am i? 
-> What is my actual level? Junior, Senior, Principal? It depends on your company or the current job position you are looking for
+  > What is my actual level? Junior, Senior, Principal? It depends on your company or the current job position you are looking for
 
-* What is my actual impact sphere? (product, team, tribe, chapter, company, market ...)
-> Understand where you can make the real difference, become noted not only by shining but helping others to do their best it's an awesome way to stand out and develop your peers
+- What is my actual impact sphere? (product, team, tribe, chapter, company, market ...)
 
-* What are my weakness and strengths?
-> Seek out for feedbacks from your peers and managers, also don't limit yourself to a unique perspective of your performance reach out for other managers, coaches and what makes sense at your current moment
+  > Understand where you can make the real difference, become noted not only by shining but helping others to do their best it's an awesome way to stand out and develop your peers
 
+- What are my weakness and strengths?
+  > Seek out for feedbacks from your peers and managers, also don't limit yourself to a unique perspective of your performance reach out for other managers, coaches and what makes sense at your current moment
 
 ### 🚩 Defining goals
 
 Based on your current moment, you should start thinking about your next goals
-> Relax this is not an easy thing to do, take a walk, watch some movies, be relaxed to understand what really warms your heart and remember: there is no perfect path, the best path is the one that suits bests for *you* on *your* current moment
 
-* Understand future goal, or most close goal to achieve (Ex: become a Sr. Engineer)
-> A common way to think about this is planning 2, 3, and 5 years in the future where you want to be?
+> Relax this is not an easy thing to do, take a walk, watch some movies, be relaxed to understand what really warms your heart and remember: there is no perfect path, the best path is the one that suits bests for _you_ on _your_ current moment
 
-* Understand the [persona](https://www.interaction-design.org/literature/topics/personas) of your closest goal
-    * Responsibilities
-    * How this person think
-    * Describe every aspect of this person based on a day a day evaluation
+- Understand future goal, or most close goal to achieve (Ex: become a Sr. Engineer)
```

Comment:
```suggestion
- Understand your future goal, or your closest goal to achieve (Ex: become a Sr. Engineer)
```


I'm not sure about this phrase though: "Understand your future goal" and "closest goal to achieve" both are talking about something that will happen in the future, so it seems pure repetition of an idea.